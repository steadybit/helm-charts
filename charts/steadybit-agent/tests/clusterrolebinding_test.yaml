templates:
  - clusterrolebinding.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: { }
