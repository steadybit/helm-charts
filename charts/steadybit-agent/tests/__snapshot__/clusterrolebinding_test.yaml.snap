manifest should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRoleBinding
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: steadybit-agent
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: steadybit-agent-0.0.0
        steadybit.com/agent: "true"
        steadybit.com/discovery-disabled: "true"
      name: RELEASE-NAME-steadybit-agent-in-NAMESPACE
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: ClusterRole
      name: RELEASE-NAME-steadybit-agent-in-NAMESPACE
    subjects:
      - kind: ServiceAccount
        name: steadybit-agent
        namespace: NAMESPACE
